### ISSUE TITLE
Allow pattern to be a module instead of a file

### ISSUE THREAD

**Issue by asottile** [Yelp/undebt#40]
Would enable a [pre-commit](http://pre-commit.com) repo that defines a bunch of modules to just have a config like:

```yaml
- id: fix-fixer-1
  entry: undebt
  language: python
  args: [-p, mypkg.fixer_1]
```

**Comment by ajm188** [comment]
I will start on this in a little bit. Should be a small change.

**Comment by ajm188** [comment]
Closed with merge of #41
